src/platform/site-wide/user-nav/sass/user-nav.scss

```diff
@@ -272,3 +273,51 @@ span.sidelines {
 #help-menu {
   width: 21rem;
 }
+.search-header-dropdown-component {
+  position: relative;
+
+  .search-dropdown-container {
+    padding: 8px 0 8px 8px;
+    max-width: 75%;
+  }
+
+  .search-dropdown-options {
+    position: absolute;
+    top: 58px;
+    background-color: $color-white;
+    width: 100%;
+    box-shadow: 0px 1px 5px 1px rgba(155, 155, 155, 0.5);
+
+    .suggestion {
+      width: 100%;
+      margin: 0;
+      padding: 8px 12px;
+      line-height: 24px;
+
+      strong {
+        font-weight: 700;
+      }
+
+      &.highlighted {
+        background-color: $color-primary;
+        color: $color-white;
+      }
+
+      &.regular {
+        color: $color-gray-dark;
+      }
+    }
+  }
+
+  .search-dropdown-input-field {
+    height: 42px;
+    display: block;
+    margin: 0;
+    max-width: 46rem;
+  }
+
+  .search-dropdown-submit-button {
+    height: 42px;
+    margin: 8px 8px 8px 8px;
+  }
+}
```

Comment:
There's a fair amount of custom CSS here. I realize it's hard to do all of this with standard design system CSS, but it might be worth having a conversation with the design team about it.

Reply:
Thanks, I'll reach out to them. I was curious for the best approach as I am inserting this component into two location with slightly differing styles.